### ISSUE TITLE
Error running tests

### ISSUE THREAD

**Issue by elsehow** [signalapp/libsignal-protocol-javascript#9]
After successfully running `grunt compile` / `grunt build`, I try to run the tests with `grunt test`, but see the following

```
Running "jshint:files" (jshint) task
>> 15 files lint free.

Running "jscs:all" (jscs) task
>> 15 files without code style errors.

Running "connect:server" (connect) task
Started connect web server on http://0.0.0.0:9998

Running "saucelabs-mocha:all" (saucelabs-mocha) task
=> Starting Tunnel to Sauce Labs
=> Stopping Tunnel to Sauce Labs
>> 
>> Error closing tunnel
```

Do contributors need to create a Sauce Labs account to run the tests? Or is something else at play?

**Comment by elsehow** [comment]
The issue is indeed that contributors need to bring their own Sauce Labs API key.

Specifically, you need to set environment variables before you run the tests, e.g.

```
SAUCE_USERNAME="your-usrname" SAUCE_ACCESS_KEY="your-access-key" grunt test
```

I will add this info to the README and do a PR

**Comment by elsehow** [comment]
see #11

**Comment by derhuerst** [comment]
Just stumbled across this: https://docs.travis-ci.com/user/sauce-connect/.

**Comment by PunkChameleon** [comment]
@elsehow Included in PR #50. Also was confused by this!
